electron-app/tests/e2e/pages/account-balances-page.ts

```diff
@@ -91,6 +92,45 @@ export class AccountBalancesPage {
     }
   }
 
+  getLocationBalances() {
+    const balanceLocations = [
+      { location: 'Blockchain', renderedValue: Zero },
+      { location: 'Banks', renderedValue: Zero },
+      { location: 'External', renderedValue: Zero },
+      { location: 'Commodities', renderedValue: Zero },
+      { location: 'Real estate', renderedValue: Zero },
+      { location: 'Equities', renderedValue: Zero }
+    ];
+
+    balanceLocations.forEach(balanceLocation => {
+      cy.get('.manual-balances-list tr').then($rows => {
+        if ($rows.text().includes(balanceLocation.location)) {
+          cy.get(
+            `.manual-balances-list tr:contains(${balanceLocation.location})`
+          ).each($row => {
+            // loops over all manual balances rows and adds up the total per location
+            // TODO: extract the replace(',', '') as to use user settings (when implemented)
+            cy.wrap($row)
+              .find(':nth-child(4) > .amount-display > .amount-display__value')
+              .then($amount => {
+                if (balanceLocation.renderedValue === Zero) {
```

Comment:
Also any way we could flatten that? this is way to nested.

Reply:
I think we can reduce the length/complexity of the `.find()` call to just `amount.display__value`, but I think we will always have to nest wrap -> find -> then. Happy to hear if you find a simpler way to do it =D